src/components/Navbar.js

```diff
@@ -0,0 +1,26 @@
+import { NavLink } from 'react-router-dom';
+
+const links = [
+  { path: '/', text: 'Home' },
+  { path: 'about', text: 'About' },
+];
+
+const Navbar = () => (
+  <nav className="navbar">
+    <ul>
+      {links.map((link) => (
+        <li key={link.text}>
+          <NavLink
+            to={link.path}
+            style={({ isActive }) => ({
+              color: isActive ? 'red' : undefined,
+            })}
+          >
+            {link.text}
+          </NavLink>
+        </li>
+      ))}
+    </ul>
+  </nav>
```

Comment:
[OPTIONAL] Please considering adding style to your navigation

Reply:
Thank you @hansderly, I'll do it!